user-guide/gpu.rst

```diff
@@ -136,23 +134,27 @@ could look like:
 
    #!/bin/bash
    #
-   #PBS -N cuda_test
-   #PBS -q gpu
-   #PBS -l select=1:ncpus=10:ngpus=1
-   #PBS -l walltime=0:20:0
-   # Budget: change 't01' to your budget code
-   #PBS -A t01
-
-   # Load the required modules (this assumes you compiled with GCC 6.3.0)
-   module load cuda
-   module load gcc/6.3.0
+   # Slurm job options (name, compute nodes, job time)
+   #SBATCH --job-name=CUDA_Example
+   #SBATCH --time=0:20:0
+   #SBATCH --partition=gpu-skylake
+   #SBATCH --qos=gpu
+   #SBATCH --exclusive
+   #SBATCH --gres=gpu:1
+   #SBATCH --nodes=1
+   #SBATCH --tasks-per-node=1
```

Comment:
Are these correct? I can believe that you need the `nodes` specifier but I thought that we were assigning cores based on how many GPUs you request so that you should **not** specify number of tasks per node (though I am not sure how this works with MPI-based GPU jobs).

Reply:
The MPI+GPU functionality needs to be investigated. Will update once I know more